pontoon/batch/utils.py

```diff
@@ -6,6 +6,16 @@
     Translation,
 )
 
+from fluent.syntax import (
+    ast,
+    FluentParser,
+    FluentSerializer,
+)
+
+
+parser = FluentParser()
+serializer = FluentSerializer()
```

Comment:
Why create these at the module level, and not in functions? Do they take so much time that it's worth optimizing it?

Reply:
Should I move them into `find_and_replace()`? They're used in `ftl_find_and_replace()`, but that's called for each string.

Reply:
Yeah they could be arguments to `ftl_find_and_replace()`.

Reply:
There's little use for passing them as arguments unless you'd like to stub them. It's safe to create them globally: they only keep state for the config (e.g. `withSpans`) and otherwise the parsing done via `FluentParser.parse` is stateless.

Reply:
But that means if you find & replace 1000 strings, you construct parser and serializer 1000 times each.

Reply:
Huh, yeah, well, whatever. It's a detail anyway, let's not sweat too much over it!